package Day0922;

public class Exception02 {
	public static void main(String[] args) {
		/*
		아래 코드들은 예외가 발생할 수 있는 코드들
		Exception
		*/
		
		try {
			int arr[] = new int [5];
			for(int i=0; i<=arr.length; i++) {
				System.out.println(arr[i]+' ');
			}
		} catch (Exception e) {
			System.out.println(e.printStackTrace());
			System.out.println("예외입니다~");
		}
		
	}
}
